cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -55,7 +55,8 @@
     {% set button_pos   = options.button_pos or 'center' %}
     {% set is_flexible  = ( options.video.width or options.video.height ) is not defined %}
     <div class="video-player
-                video-player__{{ video_player }}"
+                video-player__{{ video_player }}
+                {{ 'featured-content-module_visual' if is_flexible == false }}"
```

Comment:
ok, I think the way to go is a modifier. @KimberlyMunoz, lets go ahead and merge this and I'll fix the issue with the modifier.